extractors/python_functions/metric_detection/code_snippet.md

```diff
@@ -11,5 +12,5 @@ def metric_detection(record):
         span = quant.span
         name = quant.unit.name
 
-        yield name, span[0], span[1]
+        yield YOUR_LABEL, span[0], span[1]
```

Comment:
are these spans the same as in spacy?

Reply:
Yes, the spans are identical to spacy. However, when trying to run the code, I get: 
`ImportError: Module stemming is not installed.`
Adding `stemming==1.0.1` to the requirements.